packages/url/src/test/index.native.js

```diff
@@ -35,6 +35,18 @@ jest.mock( './fixtures/wpt-data.json', () => {
 		'file://­/p',
 		'file://%C2%AD/p',
 		'file://xn--/p',
+		'http://a.b.c.xn--pokxncvks',
+		'http://10.0.0.xn--pokxncvks',
+		'foo://ho|st/',
+		'http://ho%3Cst/',
+		'http://ho%3Est/',
+		'http://ho%7Cst/',
+		'file://%43%7C',
+		'file://%43|',
+		'file://C%7C',
+		'file://%43%7C/',
+		'https://%43%7C/',
+		'asdf://%43|/',
```

Comment:
What is the purpose of these additions?

Reply:
The implementation of URL polyfill for React Native doesn’t fully align with the one used in the browser. The text fixture we downloaded passes all checks for web but we need to list exceptions for native implementation. If the list of exceptions grows too much we might need to revisit this approach.